src/ckb/config.rs

```diff
@@ -170,6 +170,16 @@ pub struct UdtScript {
     pub args: String,
 }
 
+impl UdtScript {
+    pub fn allow_all_for_script(script: &Script) -> Self {
```

Comment:
I think it's only used in unit test code, suggest to move it to the test utils, and it should be a helper fn.

Reply:
Done in https://github.com/nervosnetwork/fiber/pull/511/commits/dac7a4185a082d2c5ef59cd6ad4b00ae47f9bc72